lib/management/index.js

```diff
@@ -13,19 +13,16 @@ const { initManagementCredentials } = require('./credentials');
 const { startWSManagementClient } = require('./push');
 const { startPollingManagementClient } = require('./poll');
 const { reshapeExceptionError } = arsenal.errorUtils;
-const { isManagementAgentUsed } = require('./agentClient');
 
 const initRemoteManagementRetryDelay = 10000;
 
+const endpointPath = 'api/v1/instance';
 const managementEndpointRoot =
     process.env.MANAGEMENT_ENDPOINT ||
     'https://api.zenko.io';
-const managementEndpoint = `${managementEndpointRoot}/api/v1/instance`;
+const managementEndpoint = `${managementEndpointRoot}/${endpointPath}`;
 
-const pushEndpointRoot =
```

Comment:
I did not see this when reviewing just the last 2 commits: I meant to keep this here as `PUSH_ENDPOINT` instead of moving to a `ORBIT_PUSH_ENDPOINT` lookup in `startWSManagementClient` and hardcoding this one here.

So to be clear the code here should read:
```js
const pushEndpointRoot = process.env.PUSH_ENDPOINT || 'https://push.api.zenko.io';
const pushEndpoint = `${pushEndpointRoot}/api/v1/instance`;
```

Reply:
done